Patches/SQLite3/SQLite3Config.cmake.in

```diff
@@ -1,9 +1,23 @@
 # Config file used to import the SQLite3 build tree.
-set( SQLite3_INCLUDE_DIR "@CMAKE_INSTALL_FULL_INCLUDEDIR@" )
-set( SQLite3_LIBRARY "@CMAKE_INSTALL_FULL_LIBDIR@/sqlite3" )
 set( SQLite3_EXECUTABLE "@CMAKE_INSTALL_FULL_BINDIR@/sqlite3_exe" )
 set( EXE_SQLITE3 "@CMAKE_INSTALL_FULL_BINDIR@/sqlite3_exe" )
 
+find_library(
+  SQLite3_LIBRARY
+  NAMES sqlite3 sqlite
+  PATHS "@CMAKE_INSTALL_FULL_LIBDIR@"
+  )
+
+find_path(
+  SQLite3_INCLUDE_DIR
+  NAMES sqlite3.h
+  PATHS "@CMAKE_INSTALL_FULL_INCLUDEDIR@"
+  )
+
```

Comment:
Probably okay, but we *do* know this path. The library is a problem beacause the file name is platform dependent (`libfoo.so`, `foo.lib`, etc.). There are a couple ways to skin that cat, but the most "correct" involves `$<TARGET_FILE>`, which would require making this a build-time artifact; `find_library` seems "good enough".